src/lib.js

```diff
@@ -108,17 +108,20 @@ function countFiles(filemap) {
         .length;
 }
 
-function deleteDir(ftp, dir) {
-    return ftp.list(dir).then(lst => {
+function deleteDir(ftp, config) {
```

Comment:
I don't follow what the merit is of passing in the config object rather than the field actually needed?